doc/docs/master/deploy-manage/deploy/deploy-w-proxy.md

```diff
@@ -553,3 +553,72 @@ The `pachyderm-proxy` service also routes Pachyderm's [**S3 gateway**](../../man
 **access Pachyderm's repo through the S3 protocol**) on port 80 (note the endpoint in the diagram below).
 
 ![Global S3 Gateway with Proxy](../../images/main-s3-gateway-with-proxy.png)
+
+## Changes to the Enterprise Server Setup
+
+Your enterprise server is [deployed in the same way as any regular cluster](#deploy-pachyderm-in-production-with-a-proxy) with a few differences (no object-store and one unique PostgreSQL database required: `dex`). The same applies when deploying an enterprise server with a proxy. 
+
+Note that the enterprise server will be deployed behind its proxy, as will each cluster registered to this enterprise server.
+
+!!! Attention
+    Enabling an embedded enterprise server with your pachd as part of the same helm installation will not work with the proxy. 
+    You can use a standalone enterprise server instead.
+
+Follow your regular [enterprise server deployment and configuration instructions](../../../enterprise/auth/enterprise-server/setup){target=_blank}, but for those few steps:
+
+- [Section 1: Deploy an enterprise server](#1-deploy-an-enterprise-server), in the values.yaml provided as examples:
+
+    - Remove the `externalService` section and replace it with `proxy`:
```

Comment:
Technically `externalService` is one level deeper (child of the `pachd` key) than `proxy` (which is at the root).